react-function-components/src/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return (
+    <button>Click Me!</button>
+  );
+}
+
+ReactDOM.render(
+  CustomButton(),
```

Comment:
When authoring React code, we never actually call our components, instead we create a react element.

```jsx
ReactDOM.render(
  <CustomButton />,
  document.querySelector('#root')
);
```